app.js

```diff
@@ -125,19 +126,19 @@ function generateRecipe(data) {
                 ${randomRecipe.ingredients.map(elmt => `
                 <li>${elmt.name} ${elmt.quantity}</li>
                 `).join('')}
-            </ul>
+            </ul
         </div>
     </div>
     `;
-    main.insertAdjacentHTML('beforeend', html3);
+    main.insertAdjacentHTML('beforeend', html3); // Comment on naming
     header.insertAdjacentHTML('beforeend', '<a href="index.html">New Search</a>');
 }
 
 //Event Listeners ---------------------------------------------------------------------------------------------------------------------
 btnSearch.addEventListener('click', () => {
-    let regEx = /[a-zA-Z]/;
+    let regEx = /[a-zA-Z]/; // Comment on regex and errors
     if(regEx.test(search.value)){
-    fetchInfo(`http://api.tvmaze.com/search/shows?q=${search.value}`)
+    fetchInfo(`http://api.tvmaze.com/search/shows?q=${search.value}`) // Comment on indentation
```

Comment:
I think this should be indented one more time to make it clear that the `fetchInfo` call is inside the `if` statement.